.github/workflows/basic-tests.yml

```
@@ -394,3 +394,57 @@ jobs:
         run: |
           breeze release-management generate-issue-content-core \
             --limit-pr-count 2 --previous-release 3.0.1 --current-release 3.0.2 --verbose
+
+  test-airflow-standalone:
+    timeout-minutes: 30
+    name: "Test Airflow standalone commands"
+    runs-on: ${{ fromJSON(inputs.runners) }}
+    env:
+      UV_VERSION: ${{inputs.uv-version}}
+      AIRFLOW_HOME: ~/airflow
+    steps:
+      - name: "Checkout ${{ github.ref }} ( ${{ github.sha }} )"
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683  # v4.2.2
+        with:
+          persist-credentials: false
+      - name: "Install uv"
+        run: curl -LsSf https://astral.sh/uv/${UV_VERSION}/install.sh | sh
+      - name: "Set up Airflow home directory"
+        run: |
+          echo "Setting AIRFLOW_HOME to $AIRFLOW_HOME"
+          mkdir -p $AIRFLOW_HOME
+      - name: "Install Airflow from current repo (simulating user installation)"
+        run: |
+          uv venv
+          source .venv/bin/activate
+          uv pip install -e ./airflow-core
```

```suggestion
          set -x
          uv pip install -e ./airflow-core
```

It will show the command executed in the output of the installation step

---

Sounds good!